includes/class-wc-admin-reports-sync.php

```diff
@@ -125,6 +125,20 @@ public static function add_regenerate_tool( $tools ) {
 		);
 	}
 
+	/**
+	 * Get the number of pending actions for reports.
+	 *
+	 * @return int
+	 */
+	public static function get_pending_actions() {
```

Comment:
I'm heading out for the rest of the week, but leaving myself a piece of feedback for later😆.  I'm thinking `get_pending_actions_count` is probably a better name for this.